docs/layouts/default.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div>
+    <nav class="navbar">
+      <div class="navbar-brand">
+        <router-link to="/" class="navbar-item" exact>
+          Vue A11Y Calendar {{ version }}
+        </router-link>
+      </div>
+
+      <router-link v-for="i in nav" :key="i.title" :to="i.to" class="navbar-item">
+        {{ i.title }}
+      </router-link>
+
+      <a target="_github" href="https://github.com/IBM/vue-a11y-calendar" class="navbar-item">
```

Comment:
When using `target`, it's recommended to use `rel="noopener noreferrer"`. Please include that.